File: API/Entity/GroundVehicle.cs
using System.ComponentModel.DataAnnotations;

namespace API.Entity
{
    public class GroundVehicle
    {
        public int GroundVehicleId { get; set; }
        public String Name { get; set; }
        public Boolean IsIdling { get; set; }
        //Create FK for VehilceTypes
        public int VehicleTypeId { get; set; }
        public VehicleType VehicleType { get; set; }
        //Create FK for Positions
        public int PositionId { get; set; }
        public Position Position { get; set; }

        public ICollection<VehicleSchedule> VehicleSchedules { get; set; }
    }
}

File: client/src/app/_models/groundVehicle.ts
import { position } from "./position";
import { vehicleType } from "./vehicleType";

export class groundVehicle
{
    groundVehicleId: number;
    name: string;
    isIdling: boolean;
    vehicleTypeId: number;
    vehicleType: vehicleType;
    positionId: number;
    position: position;
}